packages/core/src/components/navbar/navbar.md

```diff
@@ -59,7 +59,25 @@ If your application is inside a fixed-width container (instead of spanning the e
 navbar to match by wrap your navbar groups in an element with your desired `width` and `margin: 0 auto;` to horizontally
 center it.
 
-@css navbar-container
+@reactCodeExample NavbarFixedWidthExample
+
+```tsx
+<Navbar>
+    <div style={{ margin: "0 auto", width: 480 }}>{/* Add me */}
+        <NavbarGroup>
+            <NavbarHeading>Blueprint</NavbarHeading>
+        <NavbarGroup />
+        <NavbarGroup align={Alignment.END}>
+            <Button icon="home" text="Home" variant="minimal" />
+            <Button icon="document" text="Files" variant="minimal" />
+            <NavbarDivider />
+            <Button icon="user" variant="minimal" />
+            <Button icon="notifications" variant="minimal" />
+            <Button icon="cog" variant="minimal" />
+        </NavbarGroup>
+    </div>
+</Navbar>
+```
```

Comment:
Can we remove this second example? Seems to be a duplicate of the code example above it

![Screenshot 2025-03-17 at 21 38 15@2x](https://github.com/user-attachments/assets/5f9464f2-f080-4c52-9e0d-cdecfccc9baf)

Reply:
Oops 🤦